logging/performance_reliability/logging-flow-control-mechanisms.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-flow-control-mechanisms"]
+= Flow control mechanisms
+:context: logging-flow-control-mechanisms
+
+toc::[]
+
+If logs are being produced faster than they can be collected, it can be difficult to predict or control the volume of logs to an output.
+
+This can result in logs being lost, or in the event of a system outage, when log buffers build up without user control, this can cause long recovery times and high latency when the connection is restored.
+
+As an administrator, you can configure flow control mechanisms for your {logging}. You can choose to limit logging rates, or ignore some logs entirely.
```

Comment:
The feature just landed does not give you options for "ignoring" logs; they are just dropped.  We should consider excluding this comment

Reply:
Updated statement to remove "ignoring"